src/app/GitUI/SplitterManager.cs

```diff
@@ -99,6 +99,10 @@ public void RestoreFromSettings(SettingsSource settings)
                         }
                     }
                 }
+                else if (_defaultDistance.HasValue)
+                {
+                    SetSplitterDistance(_defaultDistance.Value);
```

Comment:
I think this change will make it impossible to persist the collapsed state, e.g. of the Left Panel IIRC.

Reply:
I've changed this to also check for whether there hasn't been a setting saved before. By the way, this overload is not used anywhere else at the moment.